UlKit/UlKit/CatalogViewController.swift

```diff
@@ -3,9 +3,280 @@
+        shadowSneakersView.topAnchor.constraint(equalTo: categoriesSegmentedControl.bottomAnchor, constant: 20)
+            .isActive = true
+        shadowSneakersView.widthAnchor.constraint(equalToConstant: 160).isActive = true
+        shadowSneakersView.heightAnchor.constraint(equalToConstant: 120).isActive = true
+
+        sneakersImageView.leftAnchor.constraint(equalTo: shadowSneakersView.leftAnchor).isActive = true
+        sneakersImageView.rightAnchor.constraint(equalTo: shadowSneakersView.rightAnchor).isActive = true
+        sneakersImageView.topAnchor.constraint(equalTo: shadowSneakersView.topAnchor)
+            .isActive = true
+        sneakersImageView.bottomAnchor.constraint(equalTo: shadowSneakersView.bottomAnchor).isActive = true
+
+        brendsButton.leftAnchor.constraint(equalTo: view.leftAnchor, constant: 15).isActive = true
+        brendsButton.topAnchor.constraint(equalTo: sneakersImageView.bottomAnchor, constant: 15).isActive = true
+        brendsButton.widthAnchor.constraint(equalToConstant: 345).isActive = true
+        brendsButton.heightAnchor.constraint(equalToConstant: 80).isActive = true
+
+        shoesButton.leftAnchor.constraint(equalTo: view.leftAnchor, constant: 15).isActive = true
+        shoesButton.topAnchor.constraint(equalTo: brendsButton.bottomAnchor, constant: 15).isActive = true
+        shoesButton.widthAnchor.constraint(equalToConstant: 345).isActive = true
+        shoesButton.heightAnchor.constraint(equalToConstant: 80).isActive = true
+
+        bagButton.leftAnchor.constraint(equalTo: view.leftAnchor, constant: 15).isActive = true
+        bagButton.topAnchor.constraint(equalTo: shoesButton.bottomAnchor, constant: 15).isActive = true
+        bagButton.widthAnchor.constraint(equalToConstant: 345).isActive = true
+        bagButton.heightAnchor.constraint(equalToConstant: 80).isActive = true
+    }
+
+    private func setupBarButtonItem() {
+        title = Constants.titleCatalog
+        let cameraLogo = UIBarButtonItem(
+            image: UIImage(systemName: "camera"),
+            style: .plain,
+            target: self,
+            action: #selector(callsCamera)
+        )
+        let barcodeLogo = UIBarButtonItem(
+            image: UIImage(systemName: "barcode.viewfinder"),
+            style: .plain,
+            target: self,
+            action: #selector(selectBarcode)
+        )
+        navigationItem.rightBarButtonItems = [cameraLogo, barcodeLogo]
+
+        [cameraLogo, barcodeLogo].forEach { $0.tintColor = .black }
+    }
+
+    private func createButton(title: String, imageName: String, action: Selector? = nil) -> UIButton {
+        let button = UIButton()
+        button.layer.cornerRadius = 12
+        button.setTitle(title, for: .normal)
+        button.setTitleColor(.black, for: .normal)
+        button.backgroundColor = .grayLight
+        button.titleLabel?.font = UIFont(name: Constants.fontVerdanaBold, size: 14)
```

Comment:
```suggestion
        button.titleLabel?.font = .verdanaBold(ofSize: 14)
```
вот так можно было использовать наш extension